chemical-element/chemical-element.json

```diff
@@ -13,28 +13,19 @@
         },
         "Actual": {
           "description": "The measured part of the element in percentage.",
-          "type": "number",
-          "minimum": 0,
-          "maximum": 100
+          "type": "string"
```

Comment:
even though it's not the most efficient way of doing it, you could replace minimum and maximum with a pattern ?
See [this example](https://regex101.com/r/d1Kdw5/1/)

Reply:
Good idea!

Reply:
I've added `^(?:[0-9]{1,2}(\\.\\d+)?|100)$`